src/components/sendWritable/index.js

```diff
@@ -9,4 +10,8 @@ const mapStateToProps = state => ({
   pendingTransactions: state.transactions.pending,
 });
 
-export default connect(mapStateToProps)(translate()(Send));
+const mapDispatchToProps = dispatch => ({
+  settingsUpdated: data => dispatch(settingsUpdated(data)),
```

Comment:
IMO it's not DRY that any component that uses the converter needs to have this. It would be DRYer if converter had it's own HOC.